tasks/xgettext.js

```diff
@@ -44,7 +46,7 @@ module.exports = function(grunt) {
      *                                           ...
      *                                       }
    */
-  function getMessages(content, regex, subRE, quoteRegex, quote, options) {
+  function getMessages(content, regex, subRE, quoteRegex, quote, options, fileName, messageToFilesMap) {
```

Comment:
Consider using destructured arguments here, this function signature is exceedingly long and easy to mess up.